projboard/tests/test_subject.py

```diff
@@ -7,36 +7,43 @@
 REDHAT = "redhat"
 
 
+@pytest.fixture
+def generate_subject(name='name'):
+    subject = Subject(name=name)
+    subject.save()
+    return subject
+
+
 class TestSubjectModel:
     @pytest.mark.django_db
     def test_create_subject(db):
-        subject = Subject.create_new_subject(name=SPORT)
+        subject = Subject.create_new_subject(SPORT)
```

Comment:
Consider this please 
 @pytest.mark.django_db
@pytest.fixture
def generate_subject(name='name'):
    subject = Subject(name=name)
    subject.save()
    return subject
    
    
    It is called as following:
     def test_create_subject(generate_subject):
        '''Here you have to search in database the subject'''
        
        assert generate_subject.name==<to the returned subject from db>
        Please consult others, to understand how to write the tests